types/v1/math.go

```diff
@@ -312,3 +312,25 @@ func removeBigIntFactors(d, factor *big.Int) (result *big.Int, times int32) {
 	}
 	return d, times
 }
+
+// SubstractFloat64 returns a - b , where a > b.
+func SubstractFloat64(a float64, b float64) float64 {
+	return (a - b)
```

Comment:
How do we handle the negative output ? Should negative output be treated differently ?

Reply:
we will never get negative value, we will get either greater than zero or zero